README.md

```diff
@@ -19,14 +19,188 @@ The cleaned must-gather can then be found in the `must-gather-output-cleaned` fo
 # Configuration
 
 A very basic default configuration you can supply as the above `-c` flag for OpenShift can be found
-under [examples/openshift_default.yaml](examples/openshift_default.yaml). 
-
-In any case, if you want to obfuscate your domain names (e.g. DNS entries), then you have to adjust the list of `domainNames` to include yours.
+under [examples/openshift_default.yaml](examples/openshift_default.yaml). If you want to obfuscate your domain names (e.g. DNS entries), then you have to adjust the list of `domainNames` to include yours.
 
 In case you don't need networking or SDN information in the must-gather, you can run the configuration under [examples/openshift_omit_network.yaml](examples/openshift_omit_network.yaml). 
 This will ignore the largest files that also take a long time to obfuscate.
 
-The fully supported schema defined in [JSON schema](https://json-schema.org/)
-can be found in [schema.json](pkg/schema/schema.json) along with examples and documentation. A more browsable
+The configuration is explained along examples below, a fully supported schema defined in [JSON schema](https://json-schema.org/)
+can be found in [schema.json](pkg/schema/schema.json) along with more examples and documentation. A more browsable
 alternative can be found here
-on [json-schema.app](https://json-schema.app/view/%23?url=https%3A%2F%2Fraw.githubusercontent.com%2Fopenshift%2Fmust-gather-clean%2Fmain%2Fpkg%2Fschema%2Fschema.json).
\ No newline at end of file
+on [json-schema.app](https://json-schema.app/view/%23?url=https%3A%2F%2Fraw.githubusercontent.com%2Fopenshift%2Fmust-gather-clean%2Fmain%2Fpkg%2Fschema%2Fschema.json).
+
+## Obfuscation
+
+The configured obfuscation logic will be applied to each non-omitted file and on a line-by-line basis.
+Let's start with a very minimal obfuscation example:
```

Comment:
Examples can be structurally represented as subheadings or titles instead of providing a conversational beginning.

Reply:
what would be a good title here?

Reply:
Yes. That works. _MAC address obfuscation example_ or _Example for MAC address obfuscation_ . You can pick whichever sounds better for you in the context.